pyrometheus/codegen/python.py

```diff
@@ -307,9 +313,9 @@ def get_mixture_specific_heat_cp_mass(self, temperature, mass_fractions):
         return self.gas_constant * cpmix
 
     def get_mixture_specific_heat_cv_mass(self, temperature, mass_fractions):
-        cp0_r = self.get_species_specific_heats_r(temperature) - 1.0
-        cpmix = self.get_mass_average_property(mass_fractions, cp0_r)
-        return self.gas_constant * cpmix
+        cv0_r = self.get_species_specific_heats_r(temperature) - 1.0
+        cvmix = self.get_mass_average_property(mass_fractions, cv0_r)
```

Comment:
Should we use `cv_mix` instead of `cvmix`? I prefer the former over the latter, but I can't recall how the notation throughout the rest of the template.

Reply:
I am just replacing cp -> cv in the "cv function". If you wanna change to cv_mix, we have to change cp_mix in the previous function too.

Reply:
Great, thanks!